cloud-config/cloud-config-main.yml

```diff
@@ -56,6 +56,21 @@ networks:
       subnet: (( grab terraform_outputs.services_subnet_az1 ))
       security_groups:
       - (( grab terraform_outputs.bosh_security_group ))
+- name: minion
```

Comment:
Why call this network `minion`? Unless this refactor is separating the deployment into multiple networks, all instance groups will live in the same network, which we'd probably call `kubernetes`.